tests/ocf-url_origin/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+    <head>
+        <meta charset="utf-8"/>
+        <title>Check origin value</title>
+        <script type="text/javascript">
+            const current_path    = 'EPUB/content_001.xhtml'
+            function test_url() {
+                try {
+                    document.getElementById('no_scripting').style.display = 'none';
+                    document.getElementById('scripting').style.display = 'block';
+                    document.getElementById('origin').textContent = document.location.origin;
```

Comment:
An opaque origin is [serialized as `null`](https://html.spec.whatwg.org/multipage/origin.html#ascii-serialisation-of-an-origin), so this test may not be very telling if opaque URLs are used.

Reply:
Ouch. 

I can of course catch a null value and display 'opaque' instead, but that cover all possible cases? I admit being pretty unfamiliar with this whole area...

Reply:
I have added a test for the "null" value to display "opaque", and I also use "self.origin" instead of going through the document.location.origin, because MDN claims it is a more faithful representation of the origin value. (Although it may not be relevant for a top level page, and this test is not made for iframes...)